internal/layers/layers.go

```diff
@@ -1,6 +1,8 @@
 // +build windows
 
-package hcsoci
+// This package deals with container layer mounting/unmounting for LCOW and WCOW
```

Comment:
Package level doc comments should be written like `Package layers deals with...`.

Reply:
Also no newline between the comment and the `package layers` declaration.

Reply:
Updated.